public/index.js

```diff
@@ -0,0 +1,20 @@
+const items = querySelector(.'todos')
+
+cells.forEach((cell) => {
+
+})
+
+})
+
+
+const buttn = querySelector('.completed')
```

Comment:
I would name this class `.completed-button` if it's a button, so not to be confused with say the table or list of completed items.